source/lac/scalapack.cc

```diff
@@ -915,14 +915,17 @@ std::vector<NumberType> ScaLAPACKMatrix<NumberType>::eigenpairs_symmetric_by_ind
     const bool compute_eigenvectors)
 {
   // Check validity of index limits.
-  Assert (index_limits.first < (unsigned int)n_rows,ExcIndexRange(index_limits.first,0,n_rows));
-  Assert (index_limits.second < (unsigned int)n_rows,ExcIndexRange(index_limits.second,0,n_rows));
+	AssertIndexRange(index_limits.first,static_cast<unsigned int>(n_rows));
+	AssertIndexRange(index_limits.second,static_cast<unsigned int>(n_rows));
 
-  std::pair<unsigned int,unsigned int> idx = std::make_pair(std::min(index_limits.first,index_limits.second),
-                                                            std::max(index_limits.first,index_limits.second));
+  //Assert (index_limits.first < (unsigned int)n_rows,ExcIndexRange(index_limits.first,0,n_rows));
+  //Assert (index_limits.second < (unsigned int)n_rows,ExcIndexRange(index_limits.second,0,n_rows));
```

Comment:
If you don't need this code anymore, remove it instead of commenting it. In six months, someone will look at this and will wonder why this code is commented.

Reply:
I know :)
I was just building and running tests! Didn't think you would reply that quickly!